SPECS/kernel/config

```diff
@@ -7366,6 +7365,8 @@ CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT=y
 CONFIG_SECURITY_LANDLOCK=y
 CONFIG_SECURITY_IPE=y
 CONFIG_IPE_BOOT_POLICY=""
+CONFIG_IPE_POLICY_SIG_SECONDARY_KEYRING=y
+CONFIG_IPE_POLICY_SIG_PLATFORM_KEYRING=y
```

Comment:
Both SECONDARY and PLATFORM keyring changes a result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/f87bb1279e1c0177f804b363c413a281856da12d#diff-1f7bd1d89ce31840a8be21b3700fda99d047643f5a1c614c6dbb00ec86bf9341R12

[Commit aafd2ae5]: ipe: allow secondary and platform keyrings to install/update policies